service-mac.js

```diff
@@ -0,0 +1,5 @@
+const serviceInstaller = require('./utils/service-installer');
+
+var Service = require("node-mac").Service;
+
+serviceInstaller(Service);
```

Comment:
I've heard we can delete these?

Reply:
Yeah, true. That's what @theflexbird said according to the installation process.